### PR TITLE
Adding UnqiueBot

### PR DESCRIPTION
**Pull request by UniqueBot1**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

File: data/en/bots/474358088128921608.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+avatar: null
```
Comment (Andre601) [inline data/en/bots/474358088128921608.md]: No avatar?

File: data/en/bots/474358088128921608.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<br>
+daliy or weekly updates on the bot
+<br>
+
+<br>
+The bot only really has music stuff working atm so please stand by as i work on adding alot mor stuff them extra commands work aswell
+<br>
+
+
+Credits to Twentysix26 for some code in the bot
```
Comment (lepon01) [inline data/en/bots/474358088128921608.md]: Err. No

**Comment by Stylite** [comment]
This is literally a red clone

**Comment by lepon01** [comment]
Redbot

**Pull request closed without merging**